src/components/ApplicationGroup/CreateAppGroup.tsx

```diff
@@ -212,10 +220,10 @@ export default function CreateAppGroup({ appList, selectedAppGroup, closePopup }
                         disabled={selectedAppGroup && !!selectedAppGroup.value}
                     />
 
-                    {showErrorMsg && !appGroupName && (
+                    {showErrorMsg && (!appGroupName || appGroupName.length > 30) && (
```

Comment:
can we rewrite like this !(appGroupName?.length <= 30)

Reply:
Done in bugfix PR